src/Adapter/Customer/Group/QueryHandler/GetCustomerGroupForEditingHandler.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace PrestaShop\PrestaShop\Adapter\Customer\Group\QueryHandler;
```

Comment:
ping @jolelievre, @FabienPapet I guess we cannot yet move this out from Adapter as it is based on object model, right?